### ISSUE TITLE
Missing vocalized feedback after toggling "Show More/Less" facets

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13575]
**Describe the bug**

- If the user clicks on the "Show More" button, there is no audio feedback that the content of the group has been updated with more options (checkboxes).
- Currently, if the user focuses "Show More" button, it gets vocalized, then clicking on it retains the focus and it vocalizes "Show Less", this is all good, we just need to add a description of what will happen is clicking the button, then a message with aria-live that more options were added, and then vice versa for "Show Less".

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**

1. Open any Screen Reader (VO, NVDA, JAWS).
2. Tab to "Digital Cameras > Shop All Digital Cameras" category in Main Menu.
3. Tab to facet group "Stores".
4. Tab to the "Show More" button and toggle it.
5. Listen to screen reader narration.

**Expected behavior**

- Screen Reader vocalizes: _"Show more, button, clicking on this button will add more options to the active group, (clicks), 10 more options were added to the active group, tab backward to read them or forward for the next group, Show Less, button, clicking on this button will hide options from the active group, (clicks), 10 options were hidden from the active group, tab backward to read them or forward for the next group, Show More"_
- TBD

**Screenshots**
![Screen Shot 2021-08-30 at 4 26 29 PM](https://user-images.githubusercontent.com/15113219/131400932-69b526cc-b3a4-4294-b4fa-f99570b5c713.png)
